logstash-core/lib/logstash/environment.rb

```diff
@@ -1,7 +1,27 @@
 # encoding: utf-8
 require "logstash/errors"
+require "logstash/config/cpu_core_strategy"
 
 module LogStash
+
+  DEFAULT_SETTINGS = {
+    "node.name" => Socket.gethostname,
```

Comment:
I'm with @jsvd on just using a string. I see what you're going for with constants @purbon but I think its just a little too finnicky here IMHO